show-land/sass/components/_button.scss

```diff
@@ -0,0 +1,31 @@
+.btn {
+  &:link,
+  &:visited {
+    text-transform: uppercase;
+    text-decoration: none;
+    padding: 1rem 2.5rem;
+    display: inline-block;
+    border-radius: 0.8rem;
```

Comment:
This ones like before please if you can use the variables from shared folder ($spaces list for padding)